ouroboros-consensus-test/test-consensus/Test/Consensus/MiniProtocol/LocalStateQuery/Server.hs

```diff
@@ -208,6 +208,9 @@ initLgrDB k chain = do
 
     genesisLedgerDB = LgrDB.ledgerDbWithAnchor testInitExtLedger
 
+    noopTrace :: blk -> m ()
```

Comment:
We have other uses of `const $ pure ()`. Don't we want to replace these with `noopTrace`? Or just use `const $ pure ()`

Reply:
hmm.. good question, i think it's better if we replace with `noop` helper function, otherwise someone who will read the code will have to check each time to understand "wtf is const pure unit"